### PR TITLE
Send more expressive game status to clients

### PR DESCRIPTION
**Pull request by Notabilis**
See reasoning at https://code.launchpad.net/~widelands-dev/widelands/net-online-game-status/+merge/343768

### CODE REVIEW

File: wlms/client.go [hunk 1]
```diff
@@ -41,6 +41,13 @@ const (
 	RECENTLY_DISCONNECTED
 )
 
+// The protocol versions supported by the metaserver
+const (
+	BUILD19 int = 0
+	BUILD20     = 5
+	// TRUNK = 5
```
Comment (gunchleoc) [inline wlms/client.go]: This comment can go now?

File: wlms/client.go [hunk 2]
```diff
@@ -41,6 +41,13 @@ const (
 	RECENTLY_DISCONNECTED
 )
 
+// The protocol versions supported by the metaserver
+const (
+	BUILD19 int = 0
+	BUILD20     = 5
```
Comment (gunchleoc) [inline wlms/client.go]: No "int"?

**Review by gunchleoc — APPROVED** [review]
Just 2 tiny nits. Good idea to name the version numbers :)

File: wlms/client.go (hunk 1 above)
Comment (Notabilis) [reply wlms/client.go]: I left it in on purpose since it might be an entry in the future. But yes, it can go.

File: wlms/client.go (hunk 2 above)
Comment (Notabilis) [reply wlms/client.go]: Good catch. I thought I don't need one there based on what similar structures are doing. But reading about it it seems that I should have the int after all.

File: wlms/client.go (hunk 1 above)
Comment (gunchleoc) [reply wlms/client.go]: We'd probably want to call id "BUILD21" or whichever build it is anyway, so I am in favor of getting rid of it.

**Pull request merged**